adopt-a-dog/dogs.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8">
+  <title>ADOPTADOG Dogs</title>
+  <link rel="stylesheet" type="text/css"
+  href="styles/styles.css">
+</head>
+<body>
+  <img src="adoptadog-logo.svg" alt="Adopt a dog logo">
+  <header>
+    <img src="images/christopher-harris-57366-unsplash.jpg" alt="Heading image">
+    <div>
+      <h1>ADOPTADOG</h1>
+      <h2>Seattle's premiere dog adoption agency</h2>
+    </div>
+    <div>
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="dogs.html">Dogs</a></li>
+          <li><a href="blog.html">Blog</a></li>
+        </ul>
+      </nav>
+    </div>
+  </header>
+  <main>
+     <div>
+      <article>
+        <h2>Here are all of our dogs!</h2>
+        <!-- Add CSS grid, copy and paste for different dogs -->
+        <div class= "dog-info-box">
+          Murphy<br>
```

Comment:
you'll want to give the elements in the div some sort of wrapping element, like a p tag or a header of some sort. Don't forget about the adopt button. :) Good class name. I see how this can be used over and over again.